core/orchestrator.go

```diff
@@ -101,7 +101,8 @@ func (orch *orchestrator) ProcessPayment(payment net.Payment, manifestID Manifes
 	}
 
 	if payment.TicketParams == nil {
-		return fmt.Errorf("Could not find TicketParams for payment %v", payment)
+		glog.Infof("Could not find TicketParams for payment %v", payment)
```

Comment:
I feel like we can omit this log to minimize log clutter - WDYT?

Reply:
Sure!  Do you think it's useful to log when we receive an empty payment though or is that information not that relevant?

Reply:
I think it is ok to omit that log right now as we can also infer an empty payment by the absence of any logs for receiving tickets.

Reply:
Fixed in 18b4192